crates/solver/src/main.rs

```diff
@@ -389,8 +393,8 @@ async fn main() -> ! {
             }
             TransactionStrategyArg::PublicMempool => {
                 assert!(
-                    !submission_nodes.is_empty(),
-                    "missing transaction submission nodes"
+                    submission_nodes.iter().any(|node| node.can_broadcast()),
+                    "At least one submission node that can broadcast transactions must be available"
```

Comment:
Why is that the case? Is there any solver that wants us to use a broadcast node on `goerli` of `gc`?

Reply:
Note that a node that can broadcast is our usual submission node type. This line says that at lease one node must be able to reliably get the transaction mined.
I don't think we have had any requests for GC or Görli.

Reply:
Yes, of course. I mixed up the node types. 🙈